### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by kergozh**
Add new action for counting new posts in 24 hours

### CODE REVIEW

File: .github/workflows/24HPostsCountMoarFediverse.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  # uncomment the following 2 lines so that it runs automatically every 15 minutes
+  # schedule:
+  #   - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  PostsCount:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Graphana call
```
Comment (g3rv4) [inline .github/workflows/24HPostsCountMoarFediverse.yaml]: I'd change the name to something more descriptive, something like "Retrieve number of posts indexed in the last 24 hours"

**Review by g3rv4 — COMMENTED** [review]
Thanks! would you mind changing the action name?

File: .github/workflows/24HPostsCountMoarFediverse.yaml (hunk 1 above)
Comment (kergozh) [reply .github/workflows/24HPostsCountMoarFediverse.yaml]: As you wish :-)

Comment (kergozh) [reply .github/workflows/24HPostsCountMoarFediverse.yaml]: Are you speaking about the action "name" or the step "name" or the action filename?

Comment (g3rv4) [reply .github/workflows/24HPostsCountMoarFediverse.yaml]: on line 13, it says "Graphana call", that's the one :)

**Pull request merged**